INSTRUCTIONS.md

```diff
@@ -0,0 +1,15 @@
+In order to apply all manifests from .infrastructure do:
+kubectl apply -f namespace.yml
+kubectl apply -f todoapp-pod.yml
+kubectl apply -f alpine.yml
+
+After applying the changes you can test the launched pod by making requests to it from the host machine, in order to do that first apply port forwarding as follows:
+kubectl port-forward pod/todoapp-pod 8080:8080 -n todoapp
+
+Or, alternatively send a request from anoteher pod in order to do that:
```

Comment:
There is a typo in the word 'another'. It is currently spelled as 'anoteher'. Please correct it to 'another'.